Resources/Private/Language/Search.xml

```diff
@@ -28,7 +28,7 @@
             <label index="tt_content.pi_flexform.collections">Restrict search to these collections</label>
             <label index="tt_content.pi_flexform.searchIn">Restrict search to current document or collection?</label>
             <label index="tt_content.pi_flexform.searchIn.none">none</label>
-            <label index="tt_content.pi_flexform.searchIn.document">document only</label>
+            <label index="tt_content.pi_flexform.searchIn.document">document and children only</label>
```

Comment:
We should name this "descendants" instead of "children" to make clear, that it works for more than two hierarchical levels.